sh.funkplot.Funkplot.yml

```diff
@@ -0,0 +1,22 @@
+app-id: sh.funkplot.Funkplot
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+base: "io.qt.qtwebengine.BaseApp"
+base-version: "5.15-22.08"
+command: funkplot-gui
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
```

Comment:
this won't work. I didn't say to remove the wayland socket if it support wayland. Nor to change to fallback-x11 if it doesn't.